api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -0,0 +1,101 @@
+package org.pmiops.workbench.institution;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth8.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.model.Institution;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Import;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class InstitutionServiceTest {
+  @Autowired private InstitutionService service;
+
+  @TestConfiguration
+  @Import({InstitutionServiceImpl.class})
+  static class Configuration {}
```

Comment:
If you don't have a test config you can just move the `@Import({InstitutionServiceImpl.class})` under `@DataJpaTest`